src/components/List_Products.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import { getItems, saveItem } from '../lib/api';
+
+function List_Products() {
+  const [Items, setItems] = useState(null);
+  const [ItemName, setItemName] = useState(null);
```

Comment:
Naming conventions with React states: it's a good practice usually use capital letters for classes and lowercase letters for variables. (e.g. `Person` - class, `maria` - an object, an instance of `Person`). In this case, `items` and `itemName`. Please, I would appreciate your help and change. 🙏 

More info about this here: [Methods should be in camelCase and be named for their function](https://www.educative.io/blog/best-practices-react-developer#5)

Reply:
Names changed in both spelling and casing. :)